types/react-autocomplete/index.d.ts

```diff
@@ -0,0 +1,147 @@
+// Type definitions for react-autocomplete 1.8
+// Project: https://github.com/reactjs/react-autocomplete
+// Definitions by: Lee Standen <https://github.com/lstanden>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.6
+
+import { Component, ChangeEvent, ReactNode, CSSProperties, HTMLProps } from 'react';
+
+export = Autocomplete;
+declare namespace Autocomplete {
+    interface Props {
+        /**
+         * The items to display in the dropdown menu
+         */
+        items: any[];
```

Comment:
I wonder if this should be `ReadonlyArray`, as the autocomplete component doesn't mutate the array. Also, could this somehow use generics for added type safety?

I can see if I get time to put together a PR improving these types.